spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,9 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      user = sign_in_and_2fa_user
+      loa3_sp_session
```

Comment:
I'm slightly confused, the spec already exists in `saml`, it is the test you commented on ☝️ . Do you want me to completely remove this scenario from `flow_spec` and just rely on the 'live' tests?